Data/symbol-properties/symbol-properties-database.csv

```diff
@@ -3424,3 +3424,17 @@ ftxus,WBTCUSD,crypto,WBTC/USD,USD,1,1.0,0.0001,WBTC/USD,0.0001
 ftxus,YFIBTC,crypto,YFI/BTC,BTC,1,0.00025,0.001,YFI/BTC,0.001
 ftxus,YFIUSD,crypto,YFI/USD,USD,1,5.0,0.001,YFI/USD,0.001
 ftxus,YFIUSDT,crypto,YFI/USDT,USDT,1,5.0,0.001,YFI/USDT,0.001
+
+# source https://www.interactivebrokers.com/en/index.php?f=2222&exch=nyseliffe&showcategories=FUTGRP#
+nyseliffe,MXEF,future,MSCI EMERGING MARKETS INDEX,USD,50,0.1,1
+nyseliffe,MXEA,future,MSCI EAFE Index,USD,50,0.1,1
+nyseliffe,M1EU,future,MSCI Europe NTR,USD,10,1,1
+nyseliffe,M1JP,future,The MSCI Japan NTR,USD,10,0.1,1
+nyseliffe,M1MSA,future,MSCI Emerging Markets Asia Net Total Return,USD,100,0.05,1
+nyseliffe,MXUS,future,MSCI USA Index,USD,50,0.1,1
+nyseliffe,YG,future,Mini Sized NY Gold Futures,USD,32.15,0.1,1
+nyseliffe,YI,future,Mini Sized NY Silver Future,USD,1000,0.001,1
+nyseliffe,ZG,future,Gold 100 Troy Oz,USD,100,0.1,1
+nyseliffe,ZI,future,CBOT 5000 Oz Silver Futures,USD,5000,0.001,1
```

Comment:
Cool! these futures should also be added to the `FuturesExpiryFunctions.cs` & `Futures.cs`

Reply:
Done. 

TL;DR - Good Friday is the only date which will interfere with the expiry date, hence I added code to calculate Good Friday instead of hardcoding the date in market-hours-database.json.

Long version - The NYSELIFFE exchange (which was bought over by ICE) for the added futures is only officially closed for four days of the year, New Year's day, Good Friday, Christmas and Boxing day. The index futures expire on the 3rd Friday of the contract month and metal futures on the 3rd last day of the contract month. Doing some calcs the only date that can affect the expiry date is Good Friday due to the date not being static but changing. Good Friday is deterministic and I have used the code before in other projects to calculate it effectively. This was added to FuturesExpiryUtilityFunctions for others to use as it is way more efficient than adding Good Friday date to the exchange holiday dates array. You will see it in the new commit.

Reply:
Done, added the dates for 2022 and 2023 to the market-hours-database.json